### DISCUSSION TITLE
CV charge at x % SOC

### DISCUSSION THREAD

**Discussion by fsenoner** [pybamm-team/PyBaMM#4161] [Q&A]
Dear PyBaMM Team,

I have a question regarding the CV phase at a certain SOC level.

As mentioned in other discussions, there are multiple ways to define SOC. I have chosen the following:
`sol["Discharge capacity [A.h]"].entries/cap_0`

I implemented this in my experiment, for example, to charge until 40% SOC with a current of 1C:
'Charge at 1C for 0.4 hours'

Now I would like to implement a CV phase. According to the example notebook (https://docs.pybamm.org/en/stable/source/examples/notebooks/getting_started/tutorial-5-run-experiments.html#Direct-instructions), there are options to implement a CV phase at a certain voltage. I don't know the voltage after the CC phase a priori in my experiment, so how could I implement the CV phase?

Thank you
Fabian Senoner

**Comment by valentinsulzer** [comment]
It's not possible to do this with one experiment defined up front, but you can run the CC charge, evaluate the final voltage, create a new experiment step, and start the simulation from where it left off (`sim.solve(starting_solution=cc_sol)`)

**Reply by fsenoner** [reply]
Thank you
